features/features.go

```diff
@@ -51,6 +51,10 @@ const (
 	// CheckRenewalFirst will check whether an issuance is a renewal before
 	// checking the "certificates per name" rate limit.
 	CheckRenewalFirst
+	// AdminRevokeV2 will control whether the admin-revoker uses the newer
+	// SA.RevokeAuthorizationsByDomain2 RPC or the older
+	// SA.RevokeAuthorizationsByDomain RPC.
+	AdminRevokeV2
```

Comment:
How would you feel about using the giant umbrella feature `NewAuthorizationSchema` rather than a separate feature? I don't think it's strictly necessary but I feel like it might be easier to track one giant feature than a handful of smaller flags that do different things.

Reply:
Sounds good, implemented in e7d081b.